tests/examples-without-external-providers/cloud-router-2-port-routing-protocol-connection/main.tf

```diff
@@ -1,10 +1,11 @@
 provider "equinix" {
   client_id     = var.equinix_client_id
   client_secret = var.equinix_client_secret
+  endpoint = "https://uatapi.equinix.com"
```

Comment:
Should not include UAT URL in main.tf files.

Reply:
Removed it, Thanks a lot.